### DISCUSSION TITLE
Помогите!!!

### DISCUSSION THREAD

**Discussion by WinProSoft360** [bol-van/zapret#932] [Q&A]
Раньше у меня работал войс в ДС и другие сайты (rutor и другие) а сейчас только работают Ютуб и ДС (без войса).
Че делать???

**Comment by den1xal** [comment]
то же самое. Вчера обновил запрет, дискрод грузит теперь только на линуксовом компе, к rtc не подключается. Часов пять просидел, перетыкал все возможное - все также. До обновления все было окей. Еще твиттер и ему подобная дрянь тоже прекратили загружаться. Провайдер ростелеком

**Reply by den1xal** [reply]
Еще заметил, что когда выключаю службу инет не работает, включаю - работает, бред
